templates/takeovers/_german_takeover_k8.html

```diff
@@ -0,0 +1,15 @@
+<div data-lang="all" lang="de" class="p-strip--image is-dark js-takeover {% if not default %}u-hide{% endif %}" style="background-color:#326DE6">
+    <div class="row u-vertically-center">
+        <div class="col-8">
+            <h1 style="font-weight:100;">Wofür können Sie Kubernetes einsetzen?</h1>
+            <h4 class="u-sv3">Dieses Webinar liefert Ihnen alles Wichtige zum Einstieg.</h4>
+            <p class="u-align--center u-hide--medium u-hide--large">
+                <img src="{{ ASSET_SERVER_URL }}73fe5532-business+icon.svg" alt="K8 Webinar" style="width:264px; height:227px"/>
+            </p>
+            <p class="u-no-margin--top"><a href="" class="p-button--neutral">Hier klicken um teilzunehmen</a></p>
```

Comment:
Link should go to `/engage/de/what-can-you-do-with-kubernetes?utm_medium=takeover&utm_campaign=FY19_Cloud_K8s_WBR_WhatCanYouDoWithK8sDE` as per [brief](https://docs.google.com/document/d/114ZrN2RodlgiL7m3a-P0GYp10KlgAKMfRkuwAaXxcaU/edit#)